solr/core/src/java/org/apache/solr/handler/component/QueryElevationComponent.java

```diff
@@ -370,14 +374,8 @@ private ElevationProvider loadElevationProvider(SolrCore core) throws IOExceptio
     }
     log.info("Loading QueryElevation from data dir: {}", configFileName);
 
-    XmlConfigFile cfg;
-    ZkController zkController = core.getCoreContainer().getZkController();
-    if (zkController != null) {
-      cfg = new XmlConfigFile(core.getResourceLoader(), configFileName, null, null);
-    } else {
-      InputStream is = VersionedFile.getLatestFile(core.getDataDir(), configFileName);
```

Comment:
Why did you remove the use of VersionedFile?  

I did some digging... that mechanism was used on a different feature originally https://issues.apache.org/jira/browse/SOLR-351 and it was argued that for Windows OS / file system, you can't replace an already open file.  Perhaps we don't care enough about this to bother with retaining it?  If we do remove this mechanism, we need to document it in the upgrade section of the ref guide.

Reply:
It's not removed, the block is just refactored.